miscellaneous/index.json

```diff
@@ -29,6 +29,12 @@
             "url": "https://www.udacity.com/course/computer-networking--ud436",
             "type": "course",
             "level": "everyone"
+        },
+        {
+            "title": "nand2tetris",
+            "url": "https://www.nand2tetris.org/course",
+            "type": "website",
+            "level": "beginner"
```

Comment:
Hey thanks for the PR
Can you also provide how useful you found this resource by adding your review as well?

https://github.com/developersIndia/resources/blob/master/CONTRIBUTING.md#how-to-add-a-new-resource